assets/js/components/organizations/NewOrganizationModal.jsx

```diff
@@ -52,6 +59,63 @@ class NewOrganizationModal extends Component {
           onChange={this.handleInputUpdate}
           style={{ marginBottom: 20 }}
         />
+        <Text>or</Text>
+
+        {
+          this.state.importFailed ? (
```

Comment:
we should we be resetting `importFailed` to `false` when closing the modal otherwise when I reopen the modal, it will still show the error msg
<img width="537" alt="image" src="https://user-images.githubusercontent.com/51131939/157557202-609b81fe-8305-4a29-9f68-accfa9229cb5.png">